src/server/getters/get-reporting-fees.ts

```diff
@@ -62,6 +98,128 @@ interface RepStakeResults {
+        universeAddress: marketParticipants.initialReporters[i].universe,
+        crowdsourcersAreDisavowed: false,
+        isMigrated: !marketParticipants.initialReporters[i].needsMigration,
+        isFinalized: marketParticipants.initialReporters[i].reportingState === augur.constants.REPORTING_STATE.FINALIZED,
+        crowdsourcers: [],
+        initialReporterAddress: marketParticipants.initialReporters[i].initialReporter,
+      };
+    }
+  }
+  for (i = 0; i < marketParticipants.crowdsourcers.length; i++) {
+    if (marketParticipants.crowdsourcers[i].forking && marketParticipants.crowdsourcers[i].universe === parentUniverse) {
+      let forkedMarketIsEmpty: boolean = true;
+      if (forkedMarket) {
+        for (const key in forkedMarket) {
+          if (forkedMarket.hasOwnProperty(key)) {
+            forkedMarketIsEmpty = false;
+            break;
+          }
+        }
+      }
+      if (forkedMarketIsEmpty) {
+        forkedMarket = {
+          address: marketParticipants.crowdsourcers[i].marketId,
+          universeAddress: marketParticipants.crowdsourcers[i].universe,
+          isFinalized: marketParticipants.crowdsourcers[i].reportingState === augur.constants.REPORTING_STATE.FINALIZED,
+          crowdsourcers: [{address: marketParticipants.crowdsourcers[i].crowdsourcerId, isForked: marketParticipants.crowdsourcers[i].disavowed ? true : false}],
+          initialReporter: null,
+        };
+      } else {
+        if (forkedMarket) {
+          forkedMarket.crowdsourcers.push({address: marketParticipants.crowdsourcers[i].crowdsourcerId, isForked: marketParticipants.crowdsourcers[i].disavowed ? true : false});
+        }
+      }
+    } else if (!marketParticipants.crowdsourcers[i].forking && marketParticipants.crowdsourcers[i].universe !== parentUniverse) {
+      if (!keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId]) {
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId] = {
+          address: marketParticipants.crowdsourcers[i].marketId,
+          universeAddress: marketParticipants.crowdsourcers[i].universe,
+          crowdsourcersAreDisavowed: marketParticipants.crowdsourcers[i].disavowed ? true : false,
+          isMigrated: !marketParticipants.crowdsourcers[i].needsMigration,
+          isFinalized: marketParticipants.crowdsourcers[i].reportingState === augur.constants.REPORTING_STATE.FINALIZED,
+          crowdsourcers: [marketParticipants.crowdsourcers[i].crowdsourcerId],
+          initialReporterAddress: null,
+        };
+      } else {
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId].crowdsourcersAreDisavowed = marketParticipants.crowdsourcers[i].disavowed ? true : false;
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId].crowdsourcers.push(marketParticipants.crowdsourcers[i].crowdsourcerId);
+      }
+    }
+  }
+  const nonforkedMarkets: Array<NonforkedMarket> = [];
```

Comment:
Could this be replaced with 
`const nonforkedMarkets = Object.values(keyedNonforkedMarkets)` ?

Reply:
Hmm, I tried that, but the linter was saying "Property `values` does not exists on type `ObjectConstructor`".

I was able to accomplish this using the following instead, though:
```
const nonforkedMarkets = Object.keys(keyedNonforkedMarkets).map(function(key) { return keyedNonforkedMarkets[key]; });
```

Reply:
You're right, Object.values is es2017. That line you put above looks like the next best thing.